core/src/components/popover/test/basic/popover.e2e.ts

```diff
@@ -96,67 +95,35 @@ test.describe('popover: focus trap', async () => {
       await page.keyboard.press(tabKey);
     }
 
-    let scrollTop = null;
-    let selectionStart = null;
-    let previousSelectionStart = null;
-
     await expect(innerNativeTextarea).toBeFocused();
 
-    selectionStart = await getSelectionStart(innerNativeTextarea);
-    expect(selectionStart).toBe(0);
-
     await page.keyboard.press('ArrowDown');
 
-    selectionStart = await getSelectionStart(innerNativeTextarea);
-    expect(selectionStart).toBeGreaterThan(0);
-    previousSelectionStart = selectionStart;
+    await expect(innerNativeTextarea).toBeFocused();
 
-    await page.keyboard.press('ArrowDown');
+    await page.keyboard.press('ArrowUp');
 
-    selectionStart = await getSelectionStart(innerNativeTextarea);
-    expect(selectionStart).toBeGreaterThan(previousSelectionStart);
+    await expect(innerNativeTextarea).toBeFocused();
 
     await page.keyboard.press(tabKey);
     // Checking within HTML textarea
 
-    // Reset tracking variables as the focus element has changed
-    selectionStart = null;
-    previousSelectionStart = null;
-
     await expect(vanillaTextarea).toBeFocused();
 
-    selectionStart = await getSelectionStart(vanillaTextarea);
-    expect(selectionStart).toBe(0);
-
     await page.keyboard.press('ArrowDown');
 
-    selectionStart = await getSelectionStart(vanillaTextarea);
-    expect(selectionStart).toBeGreaterThan(0);
-    previousSelectionStart = selectionStart;
+    await expect(vanillaTextarea).toBeFocused();
 
-    await page.keyboard.press('ArrowDown');
```

Comment:
I think this was supposed to be ArrowUp instead of ArrowDown a second time since the other parts of this test the other keys (Home, End, etc)